### PR TITLE
step 2: ADD A NEW SERVLET done

### PR DESCRIPTION
**Pull request by nykekana**
Hey guys! I just completed step 2 of the Community Page. It was a simple change. It worked.

### CODE REVIEW

**Review by donatKapesa — APPROVED** [review]
everything looks good

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 1]
```diff
@@ -49,15 +49,15 @@ public void init() {
...
     response.setContentType("application/json");
-
+    //first, check if the user is provided
     String user = request.getParameter("user");
 
     if (user == null || user.equals("")) {
       // Request is invalid, return empty array
       response.getWriter().println("[]");
-      return;
+      return;//stop
```
Comment (bluedragon2) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java:58]: Nit: you can remove the comment "//stop". It is already obvious what "return" does. This way you reduce clutter in the code.

File: src/main/java/com/google/codeu/servlets/UserListServlet.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * Handles fetching all users for the community page.
+ */
+@WebServlet("/user-list")
+public class UserListServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+
+    response.getOutputStream().println("this will be my user list");
```
Comment (bluedragon2) [inline src/main/java/com/google/codeu/servlets/UserListServlet.java:20]: Is this a placeholder text?

Comment (nykekana) [reply src/main/java/com/google/codeu/servlets/UserListServlet.java:20]: Yes, it was. I replaced this with code that shows the list of users.

**Pull request merged**